.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
I think it would be "all"
https://prettier.io/docs/en/options#trailing-commas

Reply:
Haha, I'm personally of the opposite opinion (i.e. I prefer not allowing trailing commas. Irrespective of preference tho, I think I'm mainly of the opinion that we should keep these settings more-or-less synced with [the dash](https://github.com/lob/dashboard-vue/blob/main/.prettierrc).

Reply:
Oh really? Lol I didn't know that actually.
I have a few reasons:
1. Less changes in Git, if you want to add to an array its a 1 line change vs. 2 (Comma + Your addition)
2. I can copy and paste things easier since I don't need to add the comma
3. I can use VS Code to re-order multi-line stuff without needing to add and remove the comma

Overall they're small things but they make it easier imo. Also `lob-api` looks to use the default (Which is `"es5"` but Prettier changed the default to `"all"` in 3.0)

Reply:
Interesting... I'm roughly of the opinion that since 2 & 3 are covered by just running format-fix (which I personally have running on save), that really comes down to preference, but I see your point re: fewer git changes. I'm fine w/ either, but I'm thinking we can keep it synced with dashboard, and change them both over if it's something we want to be deliberate about. Thoughts?